src/Symfony/Component/HttpKernel/Tests/EventListener/SessionListenerTest.php

```diff
@@ -33,6 +34,77 @@
 
 class SessionListenerTest extends TestCase
 {
+    /**
+     * @dataProvider provideSessionOptions
+     * @runInSeparateProcess
+     */
+    public function testSessionCookieOptions(array $phpSessionOptions, array $sessionOptions, array $expectedSessionOptions)
+    {
+        $session = $this->getMockBuilder(Session::class)->disableOriginalConstructor()->getMock();
+        $session->expects($this->exactly(2))->method('getUsageIndex')->will($this->onConsecutiveCalls(0, 1));
+        $session->expects($this->exactly(1))->method('getId')->willReturn('123456');
+        $session->expects($this->exactly(1))->method('getName')->willReturn('PHPSESSID');
+        $session->expects($this->exactly(1))->method('save');
+        $session->expects($this->exactly(1))->method('isStarted')->willReturn(true);
+
+        if (isset($phpSessionOptions['samesite'])) {
+            ini_set('session.cookie_samesite', $phpSessionOptions['samesite']);
```

Comment:
this currently fails on PHP 7.2 because `LAX` does only exist since PHP 7.3. The `LAX` mode is in Symfony emulated for PHP 7.2 in a special way. Maybe we can use `STRICT` for the test instead and make a tests `set_samesite_by_symfony`  to make sure `lax` mode works still correctly when set by symfony.

Reply:
done